### DISCUSSION TITLE
有没可能增加一个“漏网之鱼”或者“其他”的规则分组

### DISCUSSION THREAD

**Discussion by ly08ly** [Openwrt-Passwall/openwrt-passwall2#542] [Ideas]
理解没错的话，如果域名或者ip没有匹配任何一个分组规则的话，最后会匹配到“默认”线路。
是不是可以参考clash规则，设置一个“漏网之鱼”或者“其他”分组规则，自动接收所有未匹配域名/IP。这样可以通过调整这一个分组的连接方式，快速设置类似gfwlist或者白名单的方案。
这样默认连接设置为类“gfwlist“，访问控制对个别设备设置类“白名单”会方便很多。比现在的模式要直观和方便一些，特别是要切换节点的时候。

**Comment by caoolong** [comment]
支持

**Comment by Travel2Here** [comment]
默认不就是啥都没匹配上托底的吗？跟所谓的漏网之鱼是啥区别？

**Reply by ly08ly** [reply]
是这样的，如果我的理解没错的话，现在默认这个链接包含两个功能。一是作为选定节点的变量，其它翻墙规则选默认节点的时候，通过调整默认规则的节点，快速切换全部规则的节点。二是控制“漏网之鱼”的连接方式。
如果用”默认规则”控制“漏网之鱼”，其它的每一项翻墙规则都要手动选择相应的节点，切换的时候有些折腾。感觉两个功能还是分开作为选项更直观方便一些。

**Comment by jiejianshiwaxinge** [comment]
你可以手动创建一条漏网之鱼规则

**Comment by mycastlex** [comment]
刚才转到passwall2，也倒腾了一下发现需要“漏网之鱼”这个东西。
比如某些游戏登录时需要梯子，但登上去后其实连的是国内的服务器，一直连着梯子反而更慢。
后来发现分流下面*默认这个选项就是‘漏网之鱼’，把这个选为直连就可以实现目的了。
